services/user/Homepage/ui/src/components/nav-user.tsx

```diff
@@ -201,32 +254,20 @@ export function NavUser() {
                                             <DropdownMenuSubContent>
                                                 {connectedAccounts.map(
                                                     (connectedAccount) => (
-                                                        <DropdownMenuItem
-                                                            disabled={
-                                                                isConnectingToAccount
-                                                            }
+                                                        <AccountMenuItem
```

Comment:
We should filter out the currently logged-in user's account from the "Switch Account" list. If you want to switch to a different account, doesn't make sense to have your current account as an option. Doing that in a separate PR is also justifiable.